src/settings.js

```diff
@@ -71,7 +71,9 @@ function getSettings(cliArgs, cwd = process.cwd(), options = CLIOPTIONS) {
 		settings.outputTokens = path.normalize(`${settings.output}/tokens/`);
 	}
 
-	settings.tokensFormat = settings.tokensFormat.toLowerCase();
+	if (settings.tokensFormat) {
```

Comment:
![slack-imgs com](https://user-images.githubusercontent.com/737821/79715191-595ce200-8316-11ea-80d6-1d2681792949.png)
This is causing tests to fail because tokensFormat doesn’t exist